pkgs/by-name/m1/m1ddc/package.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, darwin
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "m1ddc";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "waydabber";
+    repo = "m1ddc";
+    rev = "v${version}";
```

Comment:
Use `finalAttrs`:

```nix
stdenv.mkDerivation (finalAttrs {
  pname = "m1ddc";
  version = "1.2.0";

  src = fetchFromGitHub {
    owner = "waydabber";
    repo = "m1ddc";
    rev = "v${finalAttrs.version}";
    sha256 = "sha256-obs2qQvSkIDsWhCXJOF1Geiqqy19KDf0InyxRVod4hk=";
  };
  ...
})
```

Reply:
And done, with a colon, that I think it was necessary. Merci pour vos corrections, @Sigmanificient !